backstack/src/commonMain/kotlin/com/slack/circuit/backstack/BackStack.kt

```diff
@@ -35,6 +35,9 @@ public interface BackStack<R : Record> : Iterable<R> {
   /** The top-most record in the [BackStack], or `null` if the [BackStack] is empty. */
   public val topRecord: R?
 
+  /** The bottom-most record in the [BackStack], or `null` if the [BackStack] is empty. */
+  public val rootRecord: R?
```

Comment:
Added this to help check the current root screen without having to iterate the whole back stack.

https://github.com/slackhq/circuit/blob/3aecc67e8fcb2b5b1937ce269da235d7795263e6/samples/navigation/src/commonMain/kotlin/com/slack/circuit/sample/navigation/ContentScaffold.kt#L66-L68